composer.json

```diff
@@ -21,7 +21,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7|^7.0",
```

Comment:
this is not correct [Semver check for ^5.7|^7.0 ](https://jubianchi.github.io/semver-check/#/^5.7|^7.0/7.0.1)
i suppose this must be changed into this - [^5.7 || ^7.0](https://jubianchi.github.io/semver-check/#/^5.7||^7.0/7.0.1)

Reply:
hmm okay. I guess the parser is lenient. do you mind making the change?

Reply:
done